### PR TITLE
People Page: change photo alt and id #60

### PR DESCRIPTION
**Pull request by Lucas-Mc**
Changes the photos on the People Page to include a more descriptive`alt` tag: `Roger Mark` -> `photo of Roger Mark`. Also changes photo `id` to be more descriptive: `Roger_Mark` -> `photo_of_Roger_Mark`. Fixes part of #60.

### CODE REVIEW

**Comment by tompollard** [comment]
I think the aim is to ensure that the image has some descriptive alt text? If so, why not insert "Photo of" into the template? This should be human readable text, so no need to strip the spaces. 

e.g. isn't this all that is needed?

```
<img class="img-fluid" width="120" src="{{ url_for('static', filename='images/' + person.image )}}" 
alt="Photo of {{ person.name }}">
```

**Comment by Lucas-Mc** [comment]
Yeah it's the same thing but I'll change it to make it easier to understand.

File: templates/people.html [hunk 1]
```diff
@@ -47,7 +47,7 @@ <h3 class="site-section-meta">{{ person.name }}</h3>
         <h4 class="font-italic">People</h4>
         <ul>
           {% for person in people.current_members %}
-            <li><a href="#{{ person.name | replace(' ','_') }}">{{ person.name }}</a></li>
+            <li><a href="#{{ person.tag_id }}">{{ person.name }}</a></li>
```
Comment (tompollard) [inline templates/people.html:50]: What is the purpose of this change?

Comment (Lucas-Mc) [reply templates/people.html:50]: It makes the link more descriptive, and now that I look at it, looks better than the custom template tags. Plus we can change it later once in the function in `main.py` instead of changing it multiple times in the template.

Comment (tompollard) [reply templates/people.html:50]: This is an internal tag to the profile of someone in the lab (not to a photo). `photo_of_X` seems less clear than using the name.
